src/drt/src/db/obj/frVia.h

```diff
@@ -102,11 +102,11 @@ class frVia : public frRef
    */
 
   dbOrientType getOrient() const override { return dbOrientType(); }
-  void setOrient(const dbOrientType& tmpOrient) override { ; }
+  void setOrient(const dbOrientType& tmpOrient) { ; }
```

Comment:
why keep it?

Reply:
Great question, I kept them just because they were already there and I wanted to make minimal changes to `frVia`, but you are right, I should remove them